docker/lite/Dockerfile

```diff
@@ -42,7 +42,7 @@ RUN /vt/dist/install_dependencies.sh mysql80
 
 # Set up Vitess user and directory tree.
 RUN groupadd -r vitess && useradd -r -g vitess vitess
-RUN mkdir -p /vt/vtdataroot && chown -R vitess:vitess /vt
+RUN mkdir -p /vt/vtdataroot /home/vitess && chown -R vitess:vitess /vt /home/vitess
```

Comment:
Sorry for coming back to this only now, but I think we should get rid of the warning generated by `mysqlsh` when it cannot find the `en_US.UTF-8` locale: `Cannot set LC_ALL to locale en_US.UTF-8: No such file or directory`.

I came up with a fix for the warning, which installs the required locale but does not set it as a default locale, which should not change the behavior of the image and Vitess. Here is the code snippet, which should be added to this file:

```Dockerfile
# Install locale required for mysqlsh
RUN apt-get update && apt-get install -y locales \
    && echo "en_US.UTF-8 UTF-8" > /etc/locale.gen \
    && locale-gen en_US.UTF-8
```

I tested locally and `mysqlsh` is no longer complaining about the locale. By doing this the image size grows by approximately 2% as seen in the logs below.

```
# Before the fix
$> docker images ls
...
vitess/lite-test              latest             0c4d123a8f79   24 hours ago    2.09GB
...
$> docker run -it --user="vitess" 0c4d123a8f79 bash
vitess@8e6183dc9343:/$ mysqlsh --version 
Cannot set LC_ALL to locale en_US.UTF-8: No such file or directory
mysqlsh   Ver 8.0.37 for Linux on x86_64 - for MySQL 8.0.37 (MySQL Community Server (GPL))
vitess@8e6183dc9343:/$

# After the fix
$> docker build -t vitess/lite-test:latest -f docker/lite/Dockerfile .
$> docker images ls
...
vitess/lite-test              latest             82391c5265a9   2 minutes ago   2.13GB
...
$> docker run -it --user="vitess" docker.io/vitess/lite-test:latest bash 
vitess@05bafcc52051:/$ mysqlsh --version 
mysqlsh   Ver 8.0.37 for Linux on x86_64 - for MySQL 8.0.37 (MySQL Community Server (GPL))
vitess@05bafcc52051:/$
```

Reply:
thanks for providing the previous patch @frouioui, it is much appreciated. I am adding this as well as part of the next commit

Reply:
This looks good to me now. Closing this comment. Thank you!